pkg/proxy/proxier.go

```diff
@@ -468,6 +468,10 @@ func (proxier *Proxier) OnUpdate(services []api.Service) {
 	glog.V(4).Infof("Received update notice: %+v", services)
 	activeServices := make(map[types.NamespacedName]bool) // use a map as a set
 	for _, service := range services {
+		// if PortalIP is "None", proxying is not required
+		if service.Spec.PortalIP == "None" {
```

Comment:
Again, it seems like we should also skip `PortalIP == ""`. It should only be possible to transition from empty string to non-empty, since we don't allow updates to PortalIP once it's set.